logic/src/commonMain/kotlin/com/wire/kalium/logic/data/event/Event.kt

```diff
@@ -101,8 +101,8 @@ sealed class Event(open val id: String) {
             val connection: Connection
         ) : User(id)
 
-        data class ClientRemove(override val id: String) : User(id)
-        data class UserDelete(override val id: String) : User(id)
+        data class ClientRemove(override val id: String, val clientId: ClientId) : User(id)
```

Comment:
at this point i think we also need to verify if the deleted client is one of mine or not (it can be for another user i'm in contact with)

Reply:
not sure! why it should send my client remove to others?